src/uu/kill/src/kill.rs

```diff
@@ -130,7 +130,7 @@ fn handle_obsolete(mut args: Vec<String>) -> (Vec<String>, Option<String>) {
     (args, None)
 }
 
-fn table() {
+fn table() -> UResult<()> {
```

Comment:
Ah yeah, sorry, my mistake. You could handle this like `print_signals`, because it can't fail.

Reply:
Yeah fixed